lib/elixir/test/elixir/kernel/quote_test.exs

```diff
@@ -307,7 +307,7 @@ defmodule Kernel.QuoteTest do
              (quote do
                 foo do
                 end
-                |> bar
+                |> bar()
                 |> baz do
```

Comment:
hi @sabiwara, I think you miss parens here and line 302. Ignore me if i'm wrong due to I'm beginner :D

Reply:
Hi @vvuunnqq , thank you for your comment 👍This is an interesting case.

Actually, the following AST

```elixir
foo 
|> bar do
  baz()
end
```

is equivalent to the following:

```elixir
foo |> bar(do: baz())
```

so it wouldn't be the same as `foo |> bar() do ...` (which wouldn't be valid anyway I think).

A usual example of this is when you pipe into an `if` or `case`:

```elixir
condition
|> if do
  baz()
end